hw/ip/csrng/data/csrng.hjson

```diff
@@ -731,15 +731,6 @@
                 This bit will stay set until the next reset.
                 '''
         }
-        { bits: "4",
```

Comment:
I realise that I'm repeating myself (sorry!), but I think it would make sense to split this commit into two pieces: the first one would remove the fifo and wire zero into the error field; the second would remove the field.